docker/httpd.conf.erb

```diff
@@ -50,10 +50,27 @@ CustomLog /dev/stdout combined
   CASValidateURL <%= ENV['CAS_VALIDATE_URL'] %>
   CASProxyValidateURL <%= ENV['CAS_PROXY_VALIDATE_URL'] %>
   CASRootProxiedAs <%= ENV['CAS_SERVICE_URL'] %>
+  <%- end -%>
+
+  <%- if ENV['OIDC_ENABLED'] == '1' -%>
```

Comment:
I'd prefer `true` instead of `1`.

Reply:
Tried to keep it consistent with the existing `CAS_ENABLED` variable.